ocaml/xapi/repository_helpers.ml

```diff
@@ -1448,13 +1445,62 @@ let get_ops_of_pending ~__context ~host ~kind =
         get_pending_guidances_of_vms ~db_get:Db.VM.get_pending_guidances
       in
       let vm_add vm value =
```

Comment:
Do we only add `RestartVM` to running/paused/suspended VMs? Also for Recommended and Full kind.

Reply:
Missed that. All of Mandatory, Recommended, and Full need to be filtered out here. A comment is added for the explanation.